DigitalOffice/Services/AuthenticationService/Models/ResultUserLoginResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace LT.DigitalOffice.AuthenticationService.Models
+{
+    public class ResultUserLoginResponse
+    {
+        public Guid UserId { get; set; }
+        public string jwtToken { get; set; }
```

Comment:
Maybe jwtToken should start with a capital letter?

Reply:
Yeah, exactly

Reply:
Fixed